src/client/api/index.ts

```diff
@@ -56,3 +65,37 @@ type GameSize = { gameSize: number }
 export const fetchGameSize = async (): Promise<number> => {
   return (await request<GameSize>(`/api/gameSize`)).gameSize
 }
+
+let token: string | null = null
```

Comment:
Erinnert mich ein bisschen an Java. Ich würde das Token evtl. immer direkt setzen? Also `localStorage.getItem('token') || null`

Reply:
zu lösen in #70